database/README.md

```diff
@@ -424,6 +424,40 @@ owners      | bytea[] | not null | surplus capturing jit order owner included in
 Indexes:
 - PRIMARY KEY: btree(`auction_id`)
 
+### jit\_orders
+
+JIT orders are orders created by solvers and settled onchain. This means that protocol is not aware of them before they appear onchain.
```

Comment:
IMO it's important to point out that these orders were not part of the auction sent to the solvers.